services/dedup/badger/badger.go

```diff
@@ -1,20 +1,23 @@
-package dedup
+package badger
 
 import (
+	"fmt"
 	"strconv"
 	"time"
 
 	"github.com/dgraph-io/badger/v4"
+	"github.com/dgraph-io/badger/v4/options"
 
 	"github.com/rudderlabs/rudder-go-kit/config"
-
+	"github.com/rudderlabs/rudder-go-kit/logger"
 	"github.com/rudderlabs/rudder-go-kit/stats"
 
 	"github.com/rudderlabs/rudder-server/rruntime"
+	"github.com/rudderlabs/rudder-server/services/dedup/types"
```

Comment:
Would it make more sense if `types` are on `dedup` package instead, do we need a separate package for them?

Reply:
It was creating a cyclic dependency